pkg/kudoctl/cmd/package_test.go

```diff
@@ -0,0 +1,51 @@
+package cmd
+
+import (
+	"os"
+	"reflect"
+	"testing"
+
+	"github.com/kudobuilder/kudo/pkg/kudoctl/files"
+	"github.com/spf13/afero"
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNewCmdBundleReturnsCmd(t *testing.T) {
+
+	newCmdBundle := newPackageCmd(fs, os.Stdout)
+
+	if newCmdBundle.Parent() != nil {
+		t.Fatal("We expect the newBundleInstall command to be returned")
+	}
+
+	sub := newCmdBundle
+	for sub.HasSubCommands() {
+		sub = sub.Commands()[0]
+	}
+
+	if reflect.ValueOf(sub.Flags().GetNormalizeFunc()).Pointer() != reflect.ValueOf(newCmdBundle.Flags().GetNormalizeFunc()).Pointer() {
+		t.Fatal("child and root commands should have the same normalization functions")
+	}
+}
+
+var bundleCmdArgs = []struct {
+	arg          []string
+	errorMessage string
+}{
+	{[]string{}, "expecting exactly one argument - directory of the operator to package"}, // 1
```

Comment:
I think we should do names for test as I try to do for all table tests. It's hard to debug then which one failed, I think numbers suck and you have to maintain them and actually go to code to figure out which case it was...

Reply:
great idea!